### PR TITLE
New Nuke OP uplink items, uplink refunding.

### PR DESCRIPTION
**Pull request by jknpj**
:cl:
 * rscadd: The nuclear operative boombox uplink can now refund uplink items that are refundable.
 * rscadd: Adds the "Modified Tank Transfer Valve" to the uplink, for 100 TC, exclusive for nuclear operatives.
 * rscadd: Nuclear operatives can now refund modified TTVs to get 15 TCs for each of them.
 * rscadd: Adds the "Syndicate-modded Combat Robot Teleporter" to the uplink, for 100 TC, exclusive for nuclear operatives.
 * rscadd: Nuclear operatives can refund their unused teleporter to get their TCs back.
 * rscadd: Adds the "Syndicate Assault Mecha - 'Mauler'" to the uplink, for 140 TC, exclusive for nuclear operatives.

### CODE REVIEW

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -243,6 +244,33 @@ var/list/uplink_items = list()
...
+	name = "Syndicate Robot Teleporter"
+	desc = "A single-use teleporter used to deploy a syndicate robot that will help with your mission. Keep in mind that unlike NT silicons these don't have access to most of the station's machinery."
+	item = /obj/item/weapon/robot_spawner/syndicate
+	refund_path = /obj/item/weapon/robot_spawner //Any spawner is fine. They are super rare and need to be unused to pass the refund check anyways.
+	cost = 50
+	refund_amount = 25
+	jobs_exclusive = list("Nuclear Operative")
+	refundable = TRUE
+
+/datum/uplink_item/dangerous/dude_bombs_lmao
```
Comment (MadmanMartian) [inline code/datums/uplink_item.dm:247]: jej

**Comment by ghost** [comment]
How will ops figure out how to use the refund feature without reading github?

**Comment by jknpj** [comment]
It's in the item description, including how many tcs they get back for refunding it.

**Comment by ghost** [comment]
Can it refund only items you buy from the uplink? Or can it also refund the already available TTVs?

Why is does the robot teleporter refund less than it costs?

**Comment by jknpj** [comment]
> Or can it also refund the already available TTVs?

Yes, you can refund the TTVs that are in your ship. 

>Why is does the robot teleporter refund less than it costs?

So you need to think carefully when picking your goodies.

**Comment by ghost** [comment]
Do wizards have to think carefully or they lose half of their spell cost?

**Comment by jknpj** [comment]
They can't refund their artifacts, so yeah, they need to think carefully.

**Comment by ghost** [comment]
I think you're being unreasonable by punishing someone needlessly, there is no unfair advantage in fully refunding an unusable robot teleporter because there are no ghosts to possess it.

It's not the ops fault nobody wants to be a robot and they get punished by losing 25 TC.

**Comment by boy2mantwicethefam** [comment]
>bomb is almost as expensive as a really powerful mech

ISHYGDDT

**Comment by SonixApache** [comment]
borg spawner should be a bit more expensive and should give a full refund if you can't get any ghosts for it

**Comment by jknpj** [comment]
Appealing to the gaymer crowd, the robot teleporter now costs 100 TC, but ca be refunded for its whole cost.

**Pull request merged**